cl/scrapers/tasks.py

```diff
@@ -248,6 +248,7 @@ def update_document_from_text(opinion):
             opinion.cluster.__dict__.update(data)
         elif model_name == "Docket":
             opinion.cluster.docket.__dict__.update(data)
+            opinion.cluster.docket.save()
```

Comment:
I was hesitant to add this line, but when I either added it later in the function where cluster and opinion are saved my tests failed.